aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -1202,5 +1206,149 @@ Tensor _sparse_csr_prod_cpu(const Tensor& input, IntArrayRef dims_to_reduce, boo
   return result;
 }
 
+std::tuple<Tensor, Tensor> _spmm_reduce_sparse_csr_cpu(
+    const Tensor& input,
+    const Tensor& weight,
```

Comment:
```suggestion
    const Tensor& other,
```

Reply:
Renamed `weight` to `other`.
Renamed `grad_input` to `grad_self`
Renamed `grad_weight` to `grad_other`